File: data/EXP01/golang_no_system_var/2420/2420.output_1.go
func goodIndices(nums []int, k int) []int {
    n := len(nums)
    result := []int{}
    
    for i := k; i < n-k; i++ {
        valid := true
        
        for j := i-1; j >= i-k; j-- {
            if nums[j] < nums[j+1] {
                valid = false
                break
            }
        }
        
        for j := i+1; j <= i+k; j++ {
            if nums[j] < nums[j-1] {
                valid = false
                break
            }
        }
        
        if valid {
            result = append(result, i)
        }
    }
    
    return result
}

File: data/EXP01/python/2420/2420.output_1.py
# @lc app=leetcode id=2420 lang=python3
from typing import List

class Solution:
    def goodIndices(self, nums: List[int], k: int) -> List[int]:
        n = len(nums)
        good_indices = []
        
        for i in range(k, n - k):
            if (
                nums[i-k:i] == sorted(nums[i-k:i], reverse=True) and
                nums[i+1:i+k+1] == sorted(nums[i+1:i+k+1])
            ):
                good_indices.append(i)
        
        return good_indices